test.scss

```diff
@@ -4,8 +4,24 @@
 
 // All errors must be fixed before a release is rolled out
 
-$space: 16px;
-
-@import 'node_modules/oleg-starter-pack/all';
+//Testing for Default
+$space: 16px; //Already included in `config/_config.space.scss`, but we can overwrite and use this value instead
```

Comment:
Adding default so this will only be applied if there is no `$space` variable set previously